tests/host-functions/plugin/plugin.go

```diff
@@ -16,6 +16,11 @@ func main() {
 
 type TestPlugin struct{}
 
+func (p TestPlugin) Close(ctx context.Context) error {
```

Comment:
if this is meant to stay in the codebase, please add more note about what should be done here. otherwise remove

Reply:
Oh, yes. My bad, fixed.